tests/entrypoints/test_openai_vision.py

```diff
@@ -0,0 +1,167 @@
+from pathlib import Path
+
+import openai
+import pytest
+import ray
+
+from ..utils import ServerRunner
+
+MODEL_NAME = "llava-hf/llava-1.5-7b-hf"
+VICUNA_CHAT_TEMPLATE = (Path(__file__).parent.parent.parent /
+                        "examples/template_vicuna.jinja")
+assert VICUNA_CHAT_TEMPLATE.exists()
+# Test different image extensions (JPG/PNG) and formats (gray/RGB/RGBA)
+TEST_IMAGE_URLS = [
+    "https://upload.wikimedia.org/wikipedia/commons/thumb/d/dd/Gfp-wisconsin-madison-the-nature-boardwalk.jpg/2560px-Gfp-wisconsin-madison-the-nature-boardwalk.jpg",
+    "https://upload.wikimedia.org/wikipedia/commons/f/fa/Grayscale_8bits_palette_sample_image.png",
+    "https://upload.wikimedia.org/wikipedia/commons/thumb/9/91/Venn_diagram_rgb.svg/1280px-Venn_diagram_rgb.svg.png",
+    "https://upload.wikimedia.org/wikipedia/commons/0/0b/RGBA_comp.png",
+]
+
+pytestmark = pytest.mark.asyncio
```

Comment:
`pytestmark` is now used for indicating the group to test (refer to `test_openai_server.py`). Please make changes accordingly, otherwise the test won't be included in CI.

Reply:
Hmm seems that the tests passed regardless.